Pd 892 investigate reporting changes including graphite (#2331)
truenas/documentation: content/SCALEUIReference/ReportingScreensSCALE.md

{{< trueimage src="/images/SCALE/23.10/ZFSReportingARCSizeResult.png" alt="ZFS ARC Size and ARC Result" id="ZFS ARC Size and ARC Result" >}}

{{< /expand >}}

## Reporting Exporters Screen

The **Reporting Exporters** screen displays any configured third party reporting exports on the system.
Exporting enables TrueNAS SCALE to send Netdata reporting metrics to another time-series database.
For more information, see the Netdata [exporting reference guide](https://learn.netdata.cloud/docs/exporting/exporting-reference).

{{< trueimage src="/images/SCALE/23.10/ReportingExportersScreen.png" alt="Reporting Exporters Screen" id="Reporting Exporters Screen" >}}

Click **Add** to open the **Add Reporting Exporter** screen.

### Add Reporting Exporter

Use the **Add Reporting Exporter** screen to configure third party reporting integrations.

{{< trueimage src="/images/SCALE/23.10/AddReportingExporter.png" alt="Add Reporting Exporter" id="Add Reporting Exporter" >}}

{{< truetable >}}
| Setting | Description |
|---------|-------------|

> | **Name** | Enter a unique name for the exporter configuration. If multiple instances are configured, each must have a distinct name. |

Reviewer: ```suggestion
| **Name** | Enter a unique name for the exporter configuration. If configuring multiple instances, give each a distinct name. |
```

| **Type** | Select the exporter type to configure. The only current supported option is **GRAPHITE**. |

> | **Enable** | Select to send reporting data to the configured exporter. Deselect to disable the explorer without removing configuration. |

Reviewer: ```suggestion
| **Enable** | Select to send reporting data to the configured exporter. Leave checkbox clear to disable the explorer without removing configuration. |
```

{{< /truetable >}}

Additional settings populate based on the selected **Type** option.

{{< expand "Graphite Settings" "v" >}}

{{< trueimage src="/images/SCALE/23.10/AddReportingExporterGraphite.png" alt="Graphite Settings" id="Graphite Settings" >}}

{{< truetable >}}
| Setting | Description |
|-----------|-------------|
| **Destination Ip** | Required. Enter the IP address of the Graphite server. |
| **Destination Port** | Required. Enter the port the Graphite server monitors. |

> | **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. |

Author: ```suggestion
| **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. <br><br>Netdata exports reporting metrics to Graphite in the format *prefix*.*hostname*.*chart*.*dimension*. |
```

> | **Hostname** | Enter the hostname to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |

Reviewer: This sentence is awkward, like you had a thought, then got distracted. Not sure how to parse it.
```suggestion
| **Hostname** | Enter the host name to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |
```

> | **Update Every** | Enter the interval to send data to the Graphite database, in seconds. Defaults to *1*. |

Reviewer: ```suggestion
| **Update Every** | Enter the number of sections for the interval to send data to the Graphite database. Defaults to *1*. |
```
